NuRadioReco/modules/efieldToVoltageConverter.py

```diff
@@ -126,13 +126,14 @@ def run(self, evt, station, det):
                     times_min.append(t0)
                     times_max.append(t0 + electric_field.get_number_of_samples() / electric_field.get_sampling_rate())
                     self.logger.debug("trace start time {}, cab_delty {}, tracelength {}".format(electric_field.get_trace_start_time(), cab_delay, electric_field.get_number_of_samples() / electric_field.get_sampling_rate()))
-        times_min = np.array(times_min)
-        times_max = np.array(times_max)
-        if times_min.min() < 0:
-            times_min -= times_min.min()
-            times_max -= times_min.min()
```

Comment:
I think this causes the problem, since after shifting times_min in l 132, the times_max is not shifted any more with this implementation. Also the padding is not included here, so that should come before as well